tests/languages/php/test.php

```diff
@@ -19,6 +19,7 @@
 $general = new General($client);
 
 $client->addHeader('Origin', 'http://localhost');
+$client->setEndpoint("http://chunked.appwrite.org/v1"); //TODO remember to remove this
```

Comment:
Let's not forget

Reply:
Not yet, we don't have the mock deployed to Appwrite.io yet

Reply:
Makes sense.